server/routers/restaurant.test.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const request = require('supertest');
+
+const restaurantRouter = require('./restaurant');
+
+describe('restaurant router', () => {
+  let app;
+
+  beforeAll(() => {
+    app = express();
+    app.use('/restaurant', restaurantRouter);
+  });
+
+  describe('GET /restaurant/:id/reservations/', () => {
+    test('should return 200 response', () => request(app).get('/restaurant/300/reservations/')
+      .then((response) => {
+        expect(response.statusCode).toBe(200);
+      }));
+
+    test('should return expected object shape', () => request(app).get('/restaurant/300/reservations/')
+      .then((response) => {
+        const responseKeys = Object.keys(response.body);
+
+        expect(responseKeys.length).toBe(2);
```

Comment:
beth - see if there is a nicer way to do this in jest if not consider other expect libraries